website/blog/2021-03-05-GAAD-React-Native-Accessibility.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The GAAD Pledge - Improving React Native Accessibility
+author: Alexandra Marlette
+authorTitle: GAAD Pledge Open Source Accessibility Community Manager for React Native
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+## Hello React Native Community,
+
+In May 2020 Facebook was the first company to take the [GAAD pledge](https://diamond.la/GAADPledge/), by doing so they committed to making accessibility a core part of the React Native open source project. Since May, Facebook has spent that time thoughtfully reviewing and documenting accessibility gaps within React Native. So far the gap analysis has surfaced 90 issues, all of which have been translated to Github issues on [React Native's Github](https://github.com/facebook/react-native/projects/15).
+
+Overall, we found that React Native APIs provide strong support for accessibility. However, we also found many core components do not yet fully utilize platform accessibility APIs and support is missing for some platform specific features.
+
+With knowledge of where the gaps exist, work can begin on closing them, the enthusiasm and diversity of contributors have always played a critical role in the development of React Native and whether a seasoned or new contributor, we're asking you to join in making React Native more accessible.
+
+To recognize contributors for their effort, when an accessibility issue is closed and attached to a pull request, contributors will get a shout out on Twitter from our community manager. Contributors whose pull requests are accepted into the codebase will be highlighted in our monthly issues update on Medium.
+
+Please join us in making React Native more accessible for everyone.
+
+### How you can help:
+
+- New contributors should read the [contribution guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md) and browse the list of 46 [good first issues](https://github.com/facebook/react-native/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22+label%3AAccessibility) in the React Native Github
+
+- Seasoned contributors should visit [the project page for Improved React Native Accessibility](https://github.com/facebook/react-native/projects/15) to see the Github issues that need their knowledge of React Native
```

Comment:
Needs a period. 

Also, I think `Seasoned` may not be the right inclusive word to use here. Instead of referring to the traits of the person who should contribute, maybe focus on the description of the issues. For example, "issues requiring a bit more effort".

Reply:
What about:

Contributors interested in issues requiring a bit more effort should visit [the project page for Improved React Native Accessibility](https://github.com/facebook/react-native/projects/15) to see the Github issues that need their knowledge of React Native.

Reply:
Sounds great!

Reply:
Made the changes and committed them to the branch. Thank you for the feedback!